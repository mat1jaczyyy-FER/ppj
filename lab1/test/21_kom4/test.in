abc = 12 // komentar
za i od 3 do 75 // komentar
// komentar
// abc = abc + 1 // komentar
 // abc = abc + 1
 a // k
 = // k
 12 // k
 az
//komentar abc
