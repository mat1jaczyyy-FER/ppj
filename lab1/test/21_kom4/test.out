IDN 1 abc
OP_PRIDRUZI 1 =
BROJ 1 12
KR_ZA 2 za
IDN 2 i
KR_OD 2 od
BROJ 2 3
KR_DO 2 do
BROJ 2 75
IDN 6 a
OP_PRIDRUZI 7 =
BROJ 8 12
KR_AZ 9 az
